include/solvers/stress_update/stress_update.h

```diff
@@ -0,0 +1,91 @@
+#ifndef MPM_STRESS_UPDATE_H_
+#define MPM_STRESS_UPDATE_H_
+
+#ifdef USE_GRAPH_PARTITIONING
+#include "graph.h"
+#endif
+
+#include "mesh.h"
+
+namespace mpm {
+
+//! StressUpdate class
+//! \brief StressUpdate base class to support different stress update schemes
+//! \tparam Tdim Dimension
+template <unsigned Tdim>
+class StressUpdate {
```

Comment:
The functions derived here are specific to StressUpdate.

Reply:
I see your though on stresses and time, I was thinking of splittling them as separate classes. Maybe `MPMScheme`?

Reply:
`MPMScheme` seems to be a nice one, but as you mention in the other PR, does that means that you are deriving from `MPM` class?

Reply:
`MPMScheme` is the base class, so it doesn't mean we are deriving from `MPM`. I do see your point, but it would be difficult to have class names which are similar.